src/pages/profile/Profile.tsx

```diff
@@ -18,11 +24,14 @@ const Profile = () => {
       try {
         setIsLoading(true);
         const userProfile = await getUserDocument(uid);
+        console.log(userProfile);
```

Comment:
This console.log can be removed, if not needed

Reply:
done